.github/workflows/continuous-integration.yml

```diff
@@ -43,7 +43,7 @@ jobs:
           - dependencies: "php-http/guzzle7-adapter"
             php-version: "8.2"
             symfony-deprecations-helper: "weak"
-          - dependencies: "php-http/guzzle7-adapter"
+          - dependencies: "php-http/guzzle7-adapter php-http/throttle-plugin"
```

Comment:
throttle plugin needs symfony 7.4 as minimum requirement. test optionally for 1.x and cleanup in 2.x